wscript

```diff
@@ -84,6 +86,54 @@ def _setup_openmp(conf):
     else:
         conf.end_msg("OpenMP found.", 'GREEN')
 
+def _ok_color(var, val):
+    if var == val:
+        return "GREEN"
+    else:
+        return "YELLOW"
+
+def _warn_color(var, val):
+    if var == val:
+        return "YELLOW"
+    else:
+        return "GREEN"
+
+def _err_color(var, val):
+    if var == val:
+        return "RED"
+    else:
+        return "GREEN"
+
+def _get_git_details(conf):
+    global APPNAME, VERSION, COMMIT, BRANCH
+
+    results = subprocess.run('git describe --dirty', shell=True, check=False, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
@plim Even though it is rather likely `git` will exist at build-time, I'd simply add it to the recipe.
```yaml
build:
     - git
```

The Conda package run-time wouldn't need to change. As for people compiling from source without Conda... It's up to them to have a working environment with `git` available on their `PATH`. Presumably they used `git` to obtain the code anyway, so this is probably a non-issue all around :)